### PR TITLE
define ref2seq and seq2ref for PairwiseAlignment

### LINKED ISSUE #231: seq2ref doesn't work with PairwiseAlignmentResult
```
LoadError: MethodError: `seq2ref` has no method matching seq2ref(::Int64, ::Bio.Align.PairwiseAlignmentResult{Int64,Bio.Seq.BioSequence{Bio.Seq.AminoAcidAlphabet},Bio.Seq.BioSequence{Bio.Seq.AminoAcidAlphabet}})
Closest candidates are:
  seq2ref(::Integer, !Matched::Bio.Align.Alignment)
  seq2ref(::Integer, !Matched::Bio.Align.AlignedSequence{S})
while loading In[52], in expression starting on line 1
```

### PR DESCRIPTION
**Pull request by bicycle1885**
This closes #231.

@diegozea Is this a reasonable solution to your issue?

### CODE REVIEW

**Comment by Ward9250** [comment]
:+1:

File: test/align/runtests.jl [hunk 1]
```diff
@@ -461,6 +461,25 @@ end
...
+        ref = dna"ATCGTATTGGT"
+        # seq:  1 A-CGTATAG-T  9
+        #         | ||||| | |
+        # ref:  1 ATCGTATTGGT 11
+        model = AffineGapScoreModel(EDNAFULL, gap_open=-4, gap_extend=-1)
+        result = pairalign(GlobalAlignment(), seq, ref, model)
+        @test isa(result, PairwiseAlignmentResult)
+        aln = alignment(result)
+        @test isa(aln, PairwiseAlignment)
+        @test seq2ref(1, aln) == (1, OP_SEQ_MATCH)
```
Comment (diegozea) [inline test/align/runtests.jl]: Would be great to test also `seq2ref(1, result)`

**Comment by diegozea** [comment]
`alignment(...)` will be needed in order to use this functions in `PairwiseAlignmentResult`. It's fine to me ;)

**Comment by bicycle1885** [comment]
Yes, `alignment` is a getter to the alignment itself.

**Pull request merged**